doc/users/next_whats_new/2020-05-tac.rst

```diff
@@ -0,0 +1,23 @@
+Add patch-work inspired API for generating grids of Axes
+--------------------------------------------------------
+
+The `.Figure` class has a method to easily generate complex
+grids of `.axes.Axes` based on text or nested list input:
+
+
+.. plot::
+   :include-source: True
+
+   axd = plt.figure(constrained_layout=True).subplot_mosaic(
+   """
```

Comment:
Could be a simpler and more realistic example, e.g.

~~~
AAE
C.E
~~~

Reply:
I made a lot of those parasite axes plots during my PhD so they feel very realistic to me, but this pattern shows off row span, column span, and a blank space so is pretty clearly better.

Reply:
was changed on the next line down.